trigger.js

```diff
@@ -105,10 +108,30 @@ class TriggerHappy {
         this._executeTriggers(triggers);
     }
 
-    _onUpdateJournal(journal) {
-        if (journal.id === this._journalId || journal.name === this.journalName)
+    _onUpdateJournal(journal, update) {
+        if (update._id === this._journalId || journal.name === this.journalName)
             this._parseJournal();
     }
+
+    _onEndUpdate(scene, userId, update) {
+        if (update["x"] === undefined && update["y"] === undefined) return true;
+        let sceneTokens = scene.data.tokens;
+        let token = sceneTokens.find(t=>t._id === update._id);
+        if (token.hidden) return true; // hidden tokens don't trigger the trigger
+        let tokenNames = sceneTokens.map(t=>t.name); 
+        const triggers = this.triggers.filter(trigger => {
+            if (trigger.trigger.constructor.name !== "Token" || !tokenNames.includes(trigger.trigger.data.name) || trigger.trigger.data.name === token.name)
+                return false;
+            // let target = canvas.tokens.get(trigger.trigger.data._id);
+            let target = sceneTokens.find(token => token.name === trigger.trigger.data.name);
+            let tokenx = (update["x"] || token.x) + token.width * scene.data.grid / 2;
+            let tokeny = (update["y"] || token.y) + token.height * scene.data.grid / 2;
+            return (target.x <= tokenx ) && (target.x + (target.width * scene.data.grid) >= tokenx)
+                && (target.y <= tokeny ) && (target.y + (target.height * scene.data.grid) >= tokeny);
+
+        });
+        return this._executeTriggers(triggers);

```

Comment:
best to explicitely `return true`, otherwise if `_executeTriggers` starts returning `false` at some point in the future for whatever reason, then this would break the updates to token moves.

Reply:
Agreed. I actually had this, but then put it the way it was to be consistent.